src/main/java/qna/service/QnaService.java

```diff
@@ -34,26 +34,44 @@ public QnaService(QuestionRepository questionRepository, AnswerRepository answer
 
     @Transactional(readOnly = true)
     public Question findQuestionById(Long id) {
-        return questionRepository.findByIdAndDeletedFalse(id)
-                .orElseThrow(NotFoundException::new);
+        return questionRepository.findByIdAndDeletedFalse(id).orElseThrow(NotFoundException::new);
```

Comment:
```suggestion
        return questionRepository.findByIdAndDeletedFalse(id)
                .orElseThrow(NotFoundException::new);
```

의미있는 기능 단위로 엔터를 넣어주면 코드 파악하기에 좋습니다.
보통은 메서드가 어떤 행위를 표현하고 있으니 객체의 메서드를 호출하는 점(.)을 기준으로 엔터를 넣어줍니다.

객체지향 생활체조
한 줄에 한 점만 찍는다.